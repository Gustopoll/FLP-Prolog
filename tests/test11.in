A B
B C
A C
D E
E F
G D
D A
