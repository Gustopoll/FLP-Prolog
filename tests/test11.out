A-B B-C D-A D-E E-F G-D
A-B A-C D-A D-E E-F G-D
A-C B-C D-A D-E E-F G-D
